src/main.rs

```diff
@@ -16,29 +16,56 @@ use tokio_core::reactor::Core;
 use name_gen::random_msg;
 
 fn main() {
-    let token = get_token(); 
-
-    let epic = random_msg();
-    let tweet = DraftTweet::new(&epic);
-    
-    let mut core = Core::new().unwrap();
-    let handle = core.handle();
-    core.run(tweet.send(&token, &handle)).unwrap();
+    let token = get_token();
+
+    if token.is_err() {
+        println!("Failed to find a .env file: {:?}", token);
+    } else {
+        let epic = random_msg();
```

Comment:
I think we should generate and print the message even if we don't successfully authenticate

Reply:
This will mean we also should only write the tweet into the log as having been used if it's successfully tweeted

Reply:
Should we split the name selection and name writing into separate functions then? I think that'd be the simplest way to do this.

Reply:
I've opened a new issue for this, #26. We'll resolve that issue in a different PR.